R/event.R

```diff
@@ -36,11 +36,12 @@ dyEvent <- function(dygraph,
   
   # create event
   event <- list()
-  event$date <- asISO8601Time(date)
-  event$label <- label
+  event$pos = asISO8601Time(date)
+  if (!missing(label) && !is.null(label)) event$label <- label
```

Comment:
My thinking is that with `label = NULL` the fact that the parameter is optional is self-documenting from the signature (this is also the idiom we use of optional parameters throughout the rest of the package).